synfig-studio/src/gui/render.cpp

```diff
@@ -439,18 +465,19 @@ RenderSettings::check_target_destination()
 		//otherwise Auto is selected
 		else
 		{
-			std::list<std::string> ext_multi_auto = {{".bmp"}, {".png"},
-					{".jpg"},{".exr"},{".ppm"}};
-	
+			std::list<std::string> ext_multi_auto = {
+				{".bmp"}, {".png"},{".jpg"},{".exr"},{".ppm"},{".webp"}
+			};
+
```

Comment:
Not your fault, but: why `std::list` instead of `std::vector`?
why the double curly brackets `{`?

Reply:
The same as before, from the original code, just reworked a bit the formatting for more lisibility.
If it works before, it will workthe same after following the same pattern.
No need to introduce bug for a simple added feature, and if a refactor can be done, itcan be done afterwards. KISS ;)